src/services/webdav/backend.rs

```diff
@@ -369,7 +372,11 @@ impl WebdavBackend {
 
         let url = format!("{}{}", self.endpoint, percent_encode_path(&p));
 
-        let mut req = Request::get(&url).header(AUTHORIZATION, &self.authorization);
+        let mut req = if self.authorization.is_empty() {
+            Request::get(&url)
+        } else {
+            Request::get(&url).header(AUTHORIZATION, &self.authorization)
```

Comment:
**suggestion**: How about using

if !self.authorization.is_empty() {
    req = req.header(xxx);
}

Reply:
That's what I wrote at first, but I noticed that `req` is immutable in `webdav_head` and `webdav_delete`, so I wrote it as what you saw now. I think it would be better to write it uniformly in webdav_*, how do you see?

Reply:
I prefer the way as I suggested.

Reply:
how about the immutable `req` in `webdav_head` and `webdav_delete`? make them to `mut` or keep the now way?

Reply:
I'm a bit confused. But let's not postpone here. Please write code in the way you like, make clippy happy, and pass all CI🤩

Reply:
done